From 40c594b647660bf91bc95fe7c9358bff7f56cf2e Mon Sep 17 00:00:00 2001
From: Hyunchul Lee <hyc.lee@gmail.com>
Date: Sat, 29 May 2021 22:46:53 +0900
Subject: [PATCH] cifsd: enclose macro variables in parenthesis
Git-commit: 40c594b647660bf91bc95fe7c9358bff7f56cf2e
Patch-mainline: v5.15-rc1
References: bsn#12
Modified-by-SEL: No


checkpatch.pl complains as the following:

CHECK: Macro argument 'fp' may be better as '(fp)' to avoid
precedence issues.

Signed-off-by: Hyunchul Lee <hyc.lee@gmail.com>
Signed-off-by: Namjae Jeon <namjae.jeon@samsung.com>
Signed-off-by: Steve French <stfrench@microsoft.com>
Signed-off-by: Guoqing Jiang <guoqing.jiang@suse.com>
---
 fs/cifsd/vfs_cache.h | 16 ++++++++--------
 1 file changed, 8 insertions(+), 8 deletions(-)

diff --git a/fs/cifsd/vfs_cache.h b/fs/cifsd/vfs_cache.h
index 823fcb257a42..635eedbd497c 100644
--- a/fs/cifsd/vfs_cache.h
+++ b/fs/cifsd/vfs_cache.h
@@ -25,14 +25,14 @@
 #define KSMBD_NO_FID		(UINT_MAX)
 #define SMB2_NO_FID		(0xFFFFFFFFFFFFFFFFULL)
 
-#define FP_FILENAME(fp)		fp->filp->f_path.dentry->d_name.name
-#define FP_INODE(fp)		d_inode(fp->filp->f_path.dentry)
-#define PARENT_INODE(fp)	d_inode(fp->filp->f_path.dentry->d_parent)
-
-#define ATTR_FP(fp) (fp->attrib_only && \
-		(fp->cdoption != FILE_OVERWRITE_IF_LE && \
-		fp->cdoption != FILE_OVERWRITE_LE && \
-		fp->cdoption != FILE_SUPERSEDE_LE))
+#define FP_FILENAME(fp)		((fp)->filp->f_path.dentry->d_name.name)
+#define FP_INODE(fp)		d_inode((fp)->filp->f_path.dentry)
+#define PARENT_INODE(fp)	d_inode((fp)->filp->f_path.dentry->d_parent)
+
+#define ATTR_FP(fp) ((fp)->attrib_only && \
+		     ((fp)->cdoption != FILE_OVERWRITE_IF_LE && \
+		      (fp)->cdoption != FILE_OVERWRITE_LE && \
+		      (fp)->cdoption != FILE_SUPERSEDE_LE))
 
 struct ksmbd_conn;
 struct ksmbd_session;
-- 
2.33.0

